.github/workflows/test.yml

```diff
@@ -18,6 +18,9 @@ jobs:
             react:
               - 'packages/react/**'
 
+      - name: Increase watchers
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+
```

Comment:
This to increase the file system watchers limit and prevent this error:

ENOSPC: System limit for number of file watchers reached